code_samples/back_office/image_editor/config/packages/image_editor.yaml

```
@@ -0,0 +1,13 @@
+ezpublish:
+    system:
+        <siteaccess>:
+            image_editor:
+                action_groups:
+                    default:
+                        id: default
+                        label: Default
+                        actions:
+                            (...)
```

```suggestion
                            # ...
```
Otherwise copy-paste will cause errors.